### DISCUSSION TITLE
How do I stop custom tardis exteriors from appearing mangled when summoned in different directions?

### DISCUSSION THREAD

**Discussion by TheBread1** [eccentricdevotion/TARDIS#638] [Q&A] [answered]
Basically just the title, I made a custom exterior using the shell room and every time I travel with the tardis, it spawns in appearing mangled with blocks that have directions if it lands in any direction apart from the one I built it in

**Comment by eccentricdevotion** [comment, marked as answer]
Screenshot of how it should look?

**Reply by TheBread1** [reply]
sure, how do I send attachments over here in github? I haven't really touched git much

**Reply by eccentricdevotion** [reply]
Should be able to just drop it into the post when you write it

**Reply by TheBread1** [reply]
![intended_look](https://user-images.githubusercontent.com/89608969/169609354-3b255314-0075-47fb-b579-cdec5cafc8da.png)
![unintended-look-1](https://user-images.githubusercontent.com/89608969/169609358-97c2b1d3-6fe0-4f10-b517-3b9dfd43f78c.png)

**Reply by TheBread1** [reply]
a while ago I got it really mangled and the trapdoors looked completely detached

**Reply by eccentricdevotion** [reply]
Cool, I will take a look when I get back to my computer

**Reply by TheBread1** [reply]
thanks mate

**Reply by eccentricdevotion** [reply]
Should be fixed in latest build 2443

**Reply by TheBread1** [reply]
thank you so much mate lol
